system/core/dynamic/HTMLHelper.cfc

```diff
@@ -573,8 +573,17 @@ Description :
 		<cfargument name="noBaseURL" 	type="boolean" 	required="false" 	default="false" hint="Defaults to false. If you want to NOT append a request's ses or html base url then set this argument to true"/>
 		<cfargument name="data"			type="struct" required="false" default="#structNew()#"	hint="A structure that will add data-{key} elements to the HTML control"/>
 		<cfscript>
-			var formBuffer	= createObject( "java", "java.lang.StringBuilder" ).init( "<form" );
-			var event 		= controller.getRequestService().getContext();
+			var formBuffer	  = createObject( "java", "java.lang.StringBuilder" ).init( "<form" );
+			var event         = controller.getRequestService().getContext();
+			var desiredMethod = '';
+
+			// Browsers can't support all the HTTP verbs, so if we passed in something
+			// besides GET or POST, we'll default to POST and save off
+			// the desired method to spoof later.
+			if (arguments.method != "GET" && arguments.method != "POST") {
```

Comment:
Hi Pete, sorry to be picky, but we have started to be stricter with our syntax guidelines.  Especially on spacing.  Can you convert this to our white space rules:

```
if( arguments.method != "GET" && arguments.method != "POST" ){
```